lib/user_info_provider.rb

```diff
@@ -0,0 +1,19 @@
+require 'net/http'
+require 'json'
+
+class UserInfoProvider
+    def real_name(user_id, team_id)
+      token = $teams[team_id][:bot_access_token]
+      real_name = "no user name provided"
+
+      url = URI("https://slack.com/api/users.info?token=#{token}&user=#{user_id}&pretty=1")
+
+      begin
```

Comment:
Yeah - It's kind of a state that I don't really know the best way to handle it, it's here in order to be fault tolerant.

It don't really know how handle Net::HTTP when the url is wrong (no idea why I didn't try...) and I know that JSON.parse will complain.

So I don't rescue anything really, the application will still saving the order with real_name for the user "no user name provided" which sound okish imo. May be a rescue could log the error?